sd-validation/test/schema-org-validation-test.js

```diff
@@ -31,31 +31,6 @@ describe('schema.org validation', () => {
     assert.equal(errors[0].message, 'Unrecognized schema.org type http://schema.org/Dog');
   });
 
-  it('reports missing required fields', async () => {
```

Comment:
i don't totally grok whats happening here swapping out the google schema to the other one but...

i'm kinda surprised we that 'required' fields isn't a thing anymore. I guess schema.org just doesn't have an opinion on whats required?

Reply:
Yeah schema.org doesn't care, but Google requires some properties to enable rich snippets.

Will bring it up with @rviscomi if we want to add extra validation for this in the future.